cloudflare_cache/cloudflare_cache.php

```diff
@@ -0,0 +1,36 @@
+<?php
+// TODO: this header should not be required...
+header('Content-Type: text/plain; charset=UTF-8');
+
+// Only purge Cloudflare cache when the live environment's cache is cleared.
+if ($_ENV['PANTHEON_ENVIRONMENT'] != 'live') {
+  die();
```

Comment:
`die` with no message is not friendly, but we don't want noise when the cache is cleared on other environments.  Maybe wrap the rest of the script in a file, and call it if the environment == 'live'.